storybook/package.json

```
@@ -4,13 +4,16 @@
   "scripts": {
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
+    "test": "test-storybook --stories-json",
+    "chromatic": "npx chromatic --exit-zero-on-changes --project-token=b1bb684c98a2",
```

Since we're using Yarn PnP we'll want to use `yarn dlx` instead of `npx` here if we can

```suggestion
    "chromatic": "yarn dlx chromatic --exit-zero-on-changes --project-token=b1bb684c98a2",
```